frontend/src/packages/notification/notification-api.package.ts

```diff
@@ -0,0 +1,23 @@
+import { toast } from 'react-toastify';
+
+import { NotificationType } from './libs/enums/enums.js';
+
+class AppNotification {
```

Comment:
And name file the same way
```suggestion
class Notification {
```